### PR TITLE
REDUCER_DISABLE option for targets

### LINKED ISSUE #801: Ability to Disable Data Reduction

### PR DESCRIPTION
**Pull request by jmthomas**
closes #801

### CODE REVIEW

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/plugin.txt [hunk 1]
```diff
@@ -39,25 +39,35 @@ VARIABLE reduced_log_retain_time 2592000
...
 <% end %>
 
 <% if include_example %>
   TARGET EXAMPLE <%= example_target_name %>
     LOG_RETAIN_TIME <%= log_retain_time %>
-    REDUCED_LOG_RETAIN_TIME <%= reduced_log_retain_time %>
+    # Disable data reduction (min, hour, day) on this target data
+    # No 'ruby reducer_microservice.rb DEFAULT__REDUCER__EXAMPLE'
+    # will appear in the the process list
+    DISABLE_REDUCER
```
Comment (ryanmelt) [inline openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/plugin.txt:61]: Change to REDUCER_DISABLE to be consistent of starting with REDUCER_

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/plugin.txt [hunk 2]
```diff
@@ -39,25 +39,35 @@ VARIABLE reduced_log_retain_time 2592000
...
+    REDUCER_MAX_CPU_UTILIZATION 50
 <% end %>
 
 <% if include_inst2 %>
   TARGET INST <%= inst2_target_name %>
     LOG_RETAIN_TIME <%= log_retain_time %>
     REDUCED_LOG_RETAIN_TIME <%= reduced_log_retain_time %>
     TLM_LOG_CYCLE_TIME 600
+    # Allow the reducer microservice to take 10% of the cpu (default 30%)
+    REDUCER_MAX_CPU_UTILIZATION 10
```
Comment (ryanmelt) [inline openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/plugin.txt:52]: This is really low for the heaviest target

**Review by ryanmelt — CHANGES_REQUESTED** [review]

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/plugin.txt (hunk 2 above)
Comment (jmthomas) [reply openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/plugin.txt:52]: I'll change it ... but speaking of that. Can the reducer know it's falling behind and raise a notification?

Comment (ryanmelt) [reply openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/plugin.txt:52]: There are cases where that would be fine though.   Like if you dropped in a whole bunch of files to process at once and it just takes a long time to work through

Comment (ryanmelt) [reply openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/plugin.txt:52]: Probably worth a notification.

**Review by ryanmelt — APPROVED** [review]

**Pull request merged**